src/nav/apm.yml

```diff
@@ -1321,6 +1317,8 @@ pages:
             path: /docs/apm/applications-menu/features/configure-request-queue-reporting
           - title: Request queue config examples
             path: /docs/apm/applications-menu/features/request-queue-server-configuration-examples
+          - title: Alert information in APM
```

Comment:
@paperclypse I hid this pending a larger IA revamp of APM, but let me know if you think this should go higher.

Reply:
I think that's fine.